src/sass/_mixins.scss

```
@@ -33,21 +33,37 @@
   width: 24px;
   height: 24px;
 
+  color: map.get(tokens.$brand, white);
+  @include light-mode {
+    color: map.get(tokens.$brand, black);
+  }
+
   display: inline-flex;
   align-items: center;
   justify-content: center;
 
-  background-color: map.get(tokens.$brand, gray);
   padding: 0.25rem;
 
   // TODO: make better
   &:hover,
   &:focus {
     cursor: pointer;
-    outline: 1px solid map.get(tokens.$colors, red);
+
+    color: darken(map.get(tokens.$brand, white), 15%);
+    @include light-mode {
+      color: lighten(map.get(tokens.$brand, black), 30%);
```

This accidentally works - there's a (deprecated) `lighten` builtin in sass. You want to `@use "functions";` and change this reference (and the others to `darken` and `lighten`):
```suggestion
      color: functions.lighten(map.get(tokens.$brand, black), 30%);
```

---

This is actually fixed by #35 which linted it =). I'll fix here.